bedrock/products/tests/test_helper_misc.py

```diff
@@ -667,6 +716,74 @@ def test_vpn_relay_bundle_12_month_total_price_ca(self):
         self.assertEqual(markup, expected)
 
 
+@override_settings(
+    FXA_ENDPOINT=TEST_FXA_ENDPOINT,
+    VPN_PRODUCT_ID=TEST_VPN_PRODUCT_ID,
+    VPN_VARIABLE_PRICING=TEST_VPN_ANALYTICS_PRICING,
+    VPN_SUBSCRIPTION_URL=TEST_VPN_SUBSCRIPTION_URL,
+)
+class TestVPNAnalytics(TestCase):
```

Comment:
Suggestion (non-blocking): could this be part of the two existing link tests in `TestVPNSubscribeLink`, as opposed to needing a new `TestVPNAnalytics` class?

Also suggest adding a test for when there's no `optional_attributes` parameter passed to the helper

Reply:
✅ ✅